app/src/main/java/io/github/polysmee/agora/VoiceCall.java

```diff
@@ -88,46 +79,46 @@ public int leaveChannel() {
 
         //fail
         return ERROR_CODE;
-    }
+    }*/
 
     /**
      * mute (unmute) local user if mute arg is set to true (false)
      * @param mute
      * @return the result of the mute method of RtcEngine (0 if success, not 0 otherwise)
      */
-    public int mute(boolean mute) {
+    /*public int mute(boolean mute) {
         int result = mRtcEngine.muteLocalAudioStream(mute);
         appointment.muteUser(new DatabaseUser(MainUserSingleton.getInstance().getId()), mute);
 
         return result;
-    }
+    }*/
 

```

Comment:
Maybe remove this file since everything is commented out.

Reply:
I'll do that as soon as I explain to Adrien everything I changed so he's not lost.